src/ir_def/check_ir.ml

```diff
@@ -560,7 +560,7 @@ let rec check_exp env (exp:Ir.exp) : unit =
            (match ts2 with
             | [] -> ()
             | _ -> error env exp.at "CPSAwait answer type error");
-           typ kr <: T.Tup [cont_typ; T.Func(T.Local, T.Returns, [], [T.catch], ts2)];
+           typ krc <: T.(Tup Construct.[cont_typ; err_contT (Tup ts2); bail_contT]);
```

Comment:
I'm worried the err_contT arity might be incorrect this way, for example if ts2 = [t2] . Might be safer to revert to original or use T.seq ts2.

Reply:
Yes, `seq` seems like the right way to do, as `err_contT` will undo it.
```suggestion
           typ krc <: T.(Tup Construct.[cont_typ; err_contT (seq ts2); bail_contT]);
```